packages/squarev2-integration/src/squarev2-payment-processor.ts

```diff
@@ -133,7 +133,7 @@ export default class SquareV2PaymentProcessor {
             'cardNumber',
             'expirationDate',
             'cvv',
-            'postalCode',
+            /* Let's assume that `postalCode` is valid unless it changes. */
```

Comment:
Do we need to have this comment? It feels pretty redundant since we are not checking for that anymore.

Reply:
I left that comment so we don't lose visibility that this field should go there. In theory, the Square team will fix the issue at some point. I can remove it if it makes noise instead.